ddprof-lib/src/main/cpp/symbols_linux.cpp

```diff
@@ -277,6 +350,26 @@ void ElfParser::loadSymbols(bool use_debug) {
   }
 }
 
+void ElfParser::addSymbol(const void *start, int length, const char *name, bool update_bounds) {
+  _cc->add(start, length, name, update_bounds);
+  if (!strcmp("_start", name)) {
```

Comment:
We could use a X macro here, and just loop over every element.

Reply:
counter.h has an example of this.

Reply:
Yep. Thanks for the tip!
Added the x-macro for better code clarity.